amplify/functions/send-email/handler.ts

```diff
@@ -0,0 +1,51 @@
+import { createMimeMessage } from 'mimetext';
+import { SESClient, SendRawEmailCommand } from '@aws-sdk/client-ses';
+import { logger } from '../../../src/utils/logger';
+import type { Schema } from '../../data/resource';
+
+const client = new SESClient({ region: 'eu-west-2' });
+
+const config = () => ({
+  senderEmail: `template-created@${process.env.NOTIFY_DOMAIN_NAME}`,
```

Comment:
Can we make this "no-reply" to make it clear that it is not monitored and to keep it generic (for potential future use cases).

Reply:
updated